CoraJ.txt

```diff
@@ -0,0 +1 @@
+Hello, I'm Cora. Three words that describe me: creative, focused, analytical. One of my favorite hobbies is cooking. I really enjoy making foods from scratch, and at one point I lived on a homestead, raising my own meats and eggs, making dairy products, and growing vegetables. Now that I'm back in the city, I miss my goats and chickens, but I do enjoy having quicker access to high quality groceries. And I actually have more time for cooking now that I'm not trying to take care of 22 acres and all those animals.
```

Comment:
That is so interesting! I lived in the country and we had various animals at times: pigs, chickens, cows, etc... But you took it to a whole different level. Even though I grew up in the country, I realized early on that I was a city boy at heart. That said, I'd like to have a little bit of property someday, if/when my wife and I can afford it.